lib/Template/CollaboraTemplateProvider.php

```diff
@@ -57,6 +57,8 @@ public function getCustomTemplates(string $mimetype): array {
 			$type = 'spreadsheet';
 		} else if (in_array($mimetype, TemplateManager::MIMES_PRESENTATIONS)) {
 			$type = 'presentation';
+		} else if (in_array($mimetype, TemplateManager::MIMES_DRAWINGS)) {
+			$type = 'drawing';
```

Comment:
```suggestion
		} elseif (in_array($mimetype, TemplateManager::MIMES_DRAWINGS)) {
			$type = 'drawing';
```
We usually use `elseif` rather than `else if`.